Source/VisualStyle.swift

```diff
@@ -105,7 +110,7 @@ extension VisualStyle {
 public class DefaultVisualStyle: VisualStyle {
 
     private let alertStyle: AlertControllerStyle
-    init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
+    public init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
```

Comment:
In other case everybody who wants to use DefaultVisualStyle I have to define init method. As result inheriting DefaultVisualStyle doesn't make sense.